src/main/java/Task.java

```diff
@@ -0,0 +1,53 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    private static Task[] taskList = new Task[100];
```

Comment:
What's the difference between having taskList in Task, as compared to having it in Duke?